style.css

```diff
@@ -17,28 +94,28 @@ body{
             color:#000000;
             background-color: #b1dfd5;
             width: 100%;
-            height:200px;
+            height:500px;
```

Comment:
Yeah, it sounds like a div would be best. The other tags have specific meanings (h1 = header 1, p = paragraph, etc.) whereas a div is just a generic container that can be styled however you want.